recipes-sota/aktualizr/aktualizr-auto-prov.bb

```diff
@@ -36,15 +36,15 @@ do_install() {
         bbwarn "OSTREE_PUSH_CREDENTIALS is ignored. Please use SOTA_PACKED_CREDENTIALS"
     fi
 
+    install -d ${D}${libdir}/sota
```

Comment:
Does that cover /var/sota?

Reply:
No, but aktualizr will create /var/sota itself if it doesn't exist.